Packages/Foundation/UIDelight/Sources/UIDelight/Modifiers/View+edgeAlignement.swift

```diff
@@ -1,9 +0,0 @@
-import SwiftUI
-
-public extension View {
-        func alignmentGuide(_ alignement: VerticalAlignment, to alignementComputed: VerticalAlignment) -> some View {
-        return self.alignmentGuide(alignement) {
-            $0[alignementComputed]
```

Comment:
This file is renamed below, but as each of these lines is changed, the similarity test fails, so it's shown as a delete+add instead of a rename...